bin/node-sass

```diff
@@ -235,17 +286,69 @@ function run(options, emitter) {
 
   if (options.watch) {
     watch(options, emitter);
+  } else if (options.directory) {
+    renderDir(options, emitter);
   } else {
     render(options, emitter);
   }
 }
 
 /**
+ * Render a file
+ *
+ * @param {String} file
+ * @param {Object} options
+ * @param {Object} emitter
+ * @api private
+ */
+function renderFile(file, options, emitter) {
+  if (path.basename(file)[0] === '_') {
+    if (options.directory) {
+      return done();
+    } else {
+      return;
+    }
+  }
+  options = getOptions([path.resolve(file)], options);
+  if (options.watch) {
+    emitter.emit('warn', '=> changed: ' + file);
+  }
+  render(options, emitter);
+}
+
+/**
+ * Render all sass files in a directory
+ *
+ * @param {Object} options
+ * @param {Object} emitter
+ * @api private
+ */
+function renderDir(options, emitter) {
+  var globPath = path.resolve(process.cwd(), options.src, globPattern(options));
```

Comment:
For clarity, shouldn't `options.src` here use `options.directory` instead?  I know they are equal, but it might read more clearly for the next person who comes along.